XtermSharp.Mac/Properties/AssemblyInfo.cs

```diff
@@ -4,20 +4,20 @@
 // Information about this assembly is defined by the following attributes. 
 // Change them to the values specific to your project.
 
-[assembly: AssemblyTitle ("XtermSharp.Mac")]
-[assembly: AssemblyDescription ("")]
-[assembly: AssemblyConfiguration ("")]
-[assembly: AssemblyCompany ("")]
-[assembly: AssemblyProduct ("")]
-[assembly: AssemblyCopyright ("${AuthorCopyright}")]
-[assembly: AssemblyTrademark ("")]
-[assembly: AssemblyCulture ("")]
+//[assembly: AssemblyTitle ("XtermSharp.Mac")]
+//[assembly: AssemblyDescription ("")]
+//[assembly: AssemblyConfiguration ("")]
+//[assembly: AssemblyCompany ("")]
+//[assembly: AssemblyProduct ("")]
+//[assembly: AssemblyCopyright ("${AuthorCopyright}")]
+//[assembly: AssemblyTrademark ("")]
+//[assembly: AssemblyCulture ("")]
 
 // The assembly version has the format "{Major}.{Minor}.{Build}.{Revision}".
 // The form "{Major}.{Minor}.*" will automatically update the build and revision,
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
-[assembly: AssemblyVersion ("1.0.*")]
+//[assembly: AssemblyVersion ("1.0.0.0")]
```

Comment:
Why not just delete the file?

Reply:
that was probably me just trying to get things to build from xtermsharp